common/coap-server/src/main/java/org/thingsboard/server/coapserver/TbCoapDtlsSettings.java

```diff
@@ -66,6 +69,12 @@ public class TbCoapDtlsSettings {
     @Value("${coap.dtls.connection_id_length:}")
     private Integer cIdLength;
 
+    @Value("${coap.dtls.max_transmission_unit:}")
```

Comment:
add default values here (1024)

Reply:
What if you need null?
I suggest leaving it as is

Reply:
Ok